packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -508,12 +565,33 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
         super.dispose();
     }
 
+    /**
+     * Gets whether to use the colors option to colorize the line.
+     */
+    get useColors() {
+        return this._useColors;

```

Comment:
useColors could be public instead as it is better for perf and we do not need getter/setter as long as there are no side effects.